### ISSUE TITLE
Create a Wiki Info Panel to unburden $:/ControlPanel and sidebar (More)

### ISSUE THREAD

**Issue by telmiger** [Jermolene/TiddlyWiki5#3716]
Making the TW user interface cleaner and leaner and more transparent for beginners should always be our goal. In TW 5.1.19 information about the wiki is available in the sidebar on the _More_ tab as well as in the control panel. As a new user I had been overwhelmed by the many tabs and sub-tabs in both places and got the idea that TW is not only highly configurable but also highly complex. So these areas could only profit from less content ...

Whats more: As a non native English reader I thought the control panel would give me control over stuff displayed there – which is not the case for information content. So why not unburden the control panel from that?

As a consequence I made a [draft of a tiddler $:/InfoPanel](https://telmiger.glitch.me/#%24%3A%2FInfoPanel) and would suggest to take this as a base for a core improvement. Ideas:

- move all information-only content from $:/ControlPanel to $:/InfoPanel (see draft)
- include all content from sidebar _More_ tab on $:/InfoPanel
- maybe hide sidebar _More_ tab from new users
- a button like the cog wheel to include in the page controls for easy access to the info panel

Maybe the suggested naming is not ideal because of the existing **tiddler info panel**? On the other hand it should be closely related to the control panel ... a **wiki info panel**.

**Comment by 00SS** [comment]
I recall quite vividly, that as a new user, once I found the `More` tab, and realized how useful its vertical tab listing was for quickly finding information, it was one place I kept visiting - to check `All` the new tiddlers I made, the `Tags` I had made, and so on.

Within a couple of months, this is where I also kept going to check on `System` tiddlers I was creating. Now days I am in that tab constantly.

I feel the `More` tab is really useful where it is right now.

As a new user on empty.html, there are only 3 buttons up top, a search bar, and 4 tabs below that. I have agreed with the suggestion in #3351 to make that 5 buttons - to include the `Home` button, and the `More Actions` button. I gave my reasoning why in that issue.

I believe empty.html is in fact already so "clean and lean" that it is scary for a new-comer. I recall - I just had no idea what to do - there was almost nothing to click on - no HELP at all in the empty.html - and until I realized I had to skip back and forth between the new TiddlyWiki I had downloaded, and tiddlywiki.com - I was struggling. And then even on tiddlywiki.com, I had no idea what to read up on to get a feel of the program.

Now that I think of it, once the Introduction (beginner) level tiddler set is ready, and if it is incorporated into tiddltwiki.com, I think if its mentioned on the `GettingStarted` tiddler - it will give a new user enough guidance to get started quickly.

As for separating the "Information" part of the Control Panel away from the "Control" part, this appears to be a wise concept. The Control Panel is getting too busy - and concentrating it on "Control Settings" seems prudent.

At first glance, items to move appear to be just the Control Panel's Info-->Advanced-->
`Editor Types` , `Loaded Modules` , `Stylesheets` , `Tiddler Fields`

Perhaps the new **$:/InfoPanel** could have a duplicate of the `More` tab as well - but I feel that would not be a reason to then remove the initially displayed `More` tab from empty.html

**Comment by AnthonyMuscio** [comment]
telmiger 

Are you suggesting the control panel has wiki wide fixed items settings, and the new info active lists that change with the content of the wiki such as lists of tiddlers?

Perhaps it could be an explore or explorer panel, and variations on search would also be helpful eg system tiddlers. plugins as they will maintain the explore name.

Maybe OOSS's objection would be less if Explore was more inviting to new users than more?

In a similar vein on the Tiddler info panel **Tools** has no place. as very little is related to the specific tiddler. I was planning to place an info panel of my own there relating to all my installed macros and plugins but now realise this would break the rule that the info on a tiddler, should be about that tiddler.

Anyway the point being that such an explorer may be a good place to make a tag extensible place for Wiki Wide active tiddler info, published along with plugins, just as plugin/macro developers can now tag items to appear in the Control panel tabs for Wiki Wide Plugin settings. I would like to take this further and encourage prolific publishers (I Intend to be) to create a brand name tab and get each plugin or macro to supply relevant info, on  the proposed Explorer panel and existing tiddler info panel. This is to try and stop the proliferation of too many tabs, for example if someone used five of my plugins/tools why have 5 tabs in each section when I can combine them into one?

In someways it would be reasonable to allow users to choose if they wanted any explore, control panel tab to appear in the sidebar tab. Initially we can just provide equivalent to the current one. Basically we need to place every existing side bar tab in Control panel or Explore and add them back to the sidebar tabs.

In relation to SideBar tabs I recently create a simple method of allowing the selection of any Sidetab to appear in the sidebar (tabbless) and hide the sidebar Tabs without closing the sidebar.

Personally I think the time is ripe for improvements, however if done smartly it will add functionality but allow a backwardly compatible presentation. Perhaps tick for new sideBar 

Regards
Tony

**Comment by 00SS** [comment]
On further thought, and in reference to #3726 - the `Stylesheets` tab would need to stay in the Control Panel's--> Info-->Advanced, since configuration can be done there by dragging to change the cascading order.

**Comment by Jermolene** [comment]
Hi @telmiger @00SS regarding the proposal in the OP to separate the informational elements of `$:/ControlPanel` out into a new `$:/InfoPanel` tiddler, I'm not convinced.

The rationale for the `$:/ControlPanel` is that it's the place you go to see the stuff belonging to the system as opposed to `$:/Manager` and `$:/TagManager` which are about the users stuff. The distinction between "things I made" and "things required by the system" seems pretty clear and strong.

This proposal introduces another split, so that users will need to think whether a setting they saw was in `$:/ControlPanel`, `$:/Manager` or `$:/InfoPanel`. Worse, the distinction between "information" and "control" isn't at all clear cut. I often want to inspect an item without changing it; I need to think not of my goal (seeing a setting), but whether that setting is modifiable so that I can figure out in which place it resides.

I think there are many improvements that we can make to `$:/ControlPanel` but I think splitting it only makes sense if there is an absolutely clear cut distinction between the two sides.

**Comment by 00SS** [comment]
@telmiger @Jermolene 
Knowing the background of the Control Panel's function makes it clearer. Thanks for that @J

Now to add to

**Comment by AnthonyMuscio** [comment]
OOSS

This is my SystemTags tiddler, tagged  $:/tags/MoreSideBar caption SysTags

Works a little differently

```
<$list filter="[all[tags]prefix[$:/tags]sort[title]]">

<$transclude tiddler="$:/core/ui/TagTemplate"/>

</$list>

;Other
<$list filter="[all[tags]prefix[$:/]!prefix[$:/tags]sort[title]]">

<$transclude tiddler="$:/core/ui/TagTemplate"/>

</$list>

<hr class="tc-untagged-separator">

{{$:/core/ui/UntaggedTemplate}}
```

Tony
